docs/content/dagster-cloud/deployment/agents/kubernetes/configuration-reference.mdx

```diff
@@ -72,13 +72,24 @@ locations:
           - name: test-volume
             config_map:
               name: test-volume-configmap
-        resources:
-          limits:
-            cpu: 100m
-            memory: 128Mi
-          requests:
-            cpu: 100m
-            memory: 128Mi
```

Comment:
took this out b/c the new thing lets you customize the resources for servers vs. runs which is much more likely to be what somebody wants i think